# note: https://github.com/ds300/patch-package/issues/142#issuecomment-661067320 for changes in package.json
# yarn patch-package @libp2p/bootstrap --exclude
diff --git a/node_modules/@libp2p/bootstrap/package.json b/node_modules/@libp2p/bootstrap/package.json
index f37f005..a48ee5c 100644
--- a/node_modules/@libp2p/bootstrap/package.json
+++ b/node_modules/@libp2p/bootstrap/package.json
@@ -28,7 +28,7 @@
   ],
   "exports": {
     ".": {
-      "types": "./src/index.d.ts",
+      "types": "./dist/src/index.d.ts",
       "import": "./dist/src/index.js"
     }
   },
